helm-charts/iqserver/templates/deployment.yaml

```diff
@@ -78,3 +101,11 @@ spec:
           secret:
             secretName: {{ template "iqserver.fullname" . }}-license
   {{- end }}
+  {{- if .Values.iq.certificates }}
+        - name: certificates-volume
+          configMap:
+            name: {{ template "iqserver.fullname" . }}-certificates
+        - name: nxiq-pv-jks
+          persistentVolumeClaim:
+            claimName: {{ template "iqserver.fullname" . }}-jks
+  {{- end }}
```

Comment:
if we're going to be processing certificates, then we'll need access to the ConfigMap (incoming) and the PVC (outgoing).